google/cloud/forseti/services/scanner/dao.py

```diff
@@ -233,7 +233,8 @@ def create(self, violations, scanner_index_id):
                 violation.get('full_name', ''),
                 violation.get('resource_data', ''),
                 violation.get('violation_data', ''),
-                violation.get('rule_name', '')
+                violation.get('rule_name', ''),
+                violation.get('resource_name', '')
```

Comment:
I know we discussed if we should add in the resource_name to the hash, and I actually don't think it's required. The violation hash was recently updated (not released) to include the violated rule, so when the next release goes out, user's findings will get updated. The resource name doesn't hurt to have here, but not required.